docs/index.md

```diff
@@ -26,7 +27,8 @@ title: Perses
 
     ---
 
-    Perses is first and foremost a dashboard tool that you can use to display a variety of observability data. Today supports Prometheus metrics & Tempo traces.
```

Comment:
I find these line breaks annoying honestly, is it caused by your formatter?

Reply:
yeah that's because of my formatter. It avoids to scroll horizontally to read it

Reply:
Have you considered this more responsive option ![image](https://github.com/user-attachments/assets/4a2c47d3-e93b-47aa-923d-720f660da8e7) 😜 

Anyway I wont block the PR for that.